udata/models/owned.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+from blinker import signal
+
```

Comment:
Same coding style used in every module: import grouped by packages if not builtin.